Core/include/Acts/Vertexing/ImpactPointEstimator.ipp

```diff
@@ -225,13 +227,14 @@ Acts::ImpactPointEstimator<input_track_t, propagator_t, propagator_options_t>::
   // Set new phi value
   phi = *res;
 
+  double cosPhi = std::cos(phi);
+  double sinPhi = std::sin(phi);
+
   // Set momentum direction
-  momDir = Vector3D(std::sin(theta) * std::cos(phi),
-                    std::sin(phi) * std::sin(theta), std::cos(theta));
```

Comment:
I remember there was once a `sincos(..)` intrinsic function that does both ... probably worth checking?

Reply:
I've never heard of this but apparently it does not exist:
https://stackoverflow.com/a/24328768/7835186

Reply:
Note that GCC performs the (sin(x), cos(x)) -> (sincos(x)) transform automatically at higher optimization levels. Clang, unfortunately, does not do so unless the `-ffast-math` flag is on, most likely because sincos has slightly different numerical results on some libm versions. See https://godbolt.org/z/ye3V5f .

Reply:
Ah, ok - no worries then. Let's leave it as is.